modules/core/test/v2/unit/recovery/blockchairApi.ts

```diff
@@ -0,0 +1,96 @@
+import * as nock from 'nock';
+import * as should from 'should';
+import { Environments } from '../../../../src/v2/environments';
+import { TestBitGo } from '../../../lib/test_bitgo';
+import { BlockchairApi } from '../../../../src/v2/recovery/blockchairApi';
+
+const coinNames = ['bitcoin', 'bitcoin-sv'];
+
+function nockBlockchair(env, coinName) {
+  const baseUrl = BlockchairApi.getBaseUrl(env, coinName);
+  nock(baseUrl)
+    .get('/dashboards/address/2N7kMMaUjmBYCiZqQV7GDJhBSnJuJoTuBws?key=my____ApiKey') // unspent
+    .reply(200, {
+      data: {
+        '2N7kMMaUjmBYCiZqQV7GDJhBSnJuJoTuBws': {
+          address: {
+            type: 'scripthash',
+            script_hex: 'a9149f13f940a9461ac6e5393859faca8c513f93cd6e87',
+            balance: 20000,
+            balance_usd: 0,
+            received: 20000,
+            received_usd: 0,
+            spent: 20000,
+            spent_usd: 0,
+            output_count: 1,
+            unspent_output_count: 0,
+            first_seen_receiving: '2019-01-16 23:52:45',
+            last_seen_receiving: '2019-01-16 23:52:45',
+            first_seen_spending: '2019-01-17 02:27:18',
+            last_seen_spending: '2019-01-17 02:27:18',
+            scripthash_type: 'multisig_2_of_3',
+            transaction_count: 2,
+          },
+          transactions: [
+            '0f58644f28726159e833c2b4dbf7a46be2c0eb7f8d36c244bca765b05113880a',
+            '9a57cdf7a8ce94c1cdad90f639fd8dcab8d20f68a117a7c30dbf468652fbf7e0',
+          ],
+          utxo: [
+            {
+              block_id: 643436,
+              transaction_hash: '9a57cdf7a8ce94c1cdad90f639fd8dcab8d20f68a117a7c30dbf468652fbf7e0',
+              index: 0,
+              value: 20000,
+            },
+          ],
+        },
+      }
+    });
+}
+
+describe('blockchair api', function() {
+  let bitgo;
+  let env;
+  let apiKey;
+
+  before(() => {
```

Comment:
No need to put this in a `before()`, since you're not doing any async work - you can just assign directly in the body of the `describe`

Reply:
I agree but I need to point out one difference: if any of the calls in the `before()` block throw an exception, the test suite continues

if the it happens in `describe()` context while test suite stops

the advantage is very minor so I also think it should be outside as well